components/Terminal/index.vue

```diff
@@ -111,31 +112,24 @@ function handleInput(event: KeydownEvent) {
   inputBuffer.value = activeLineBuffer.value;
 }
 
-function handleClear(event: KeydownEvent) {
-  event.preventDefault()
-  clearScreen()
-  // HACK
-  // Unfortunately the main keydown handler function `handleInput` also captures the `l` that was inputted and just displays it to the screen
-  // to get rid of the unnecessary `l`, we clear it shortly after it has been registered
-  setTimeout(() => {
-    activeLineBuffer.value = activeLineBuffer.value.slice(0, activeLineBuffer.value.length - 1)
-  }, 50);
-}
-
 function clearScreen() {
-  history.value = []
+  history.value = [];
 }
 
-register('clear', (state, params) => {
-  // HACK
-  // we can't clear it immediately, because the 'clear' command will be drawn on screen AFTER this has run, due to the way the command systems works
-  setTimeout(clearScreen, 50);
-  return ''
-})
+register({
```

Comment:
Can we remove this event registry now its not needed?

Reply:
no, because this is the command, not the keybinding. a settimeout is still needed to make sure that things happen in the right order

Reply:
Hadn't realised this, thanks for the clarification